proposal/color-4-new-spaces-js.d.ts.md

```diff
@@ -403,7 +401,7 @@ as the result of changing some of [`internal`]'s components.
 
 * Let `changedValue` be a function that takes a string argument for `channel`
   and calls the procedure [`Changing a Component Value`] with `changes` and
-  `this` as `initial`.
+  `color` as `initial`.
```

Comment:
@jamesnw I think this was a typo -- can you confirm?

Reply:
That's correct- it needs to be the `color` in the specified `space`, and not the original color.